blog/_posts/2024-12-02-2nd-hackathon.md

```diff
@@ -0,0 +1,76 @@
+---
+ title: "A look back at the 2nd ROOT Hackathon: Python, Docs, Tutorials"
+ layout: archive
+ author: Monica Dessole
+---
+
+
+You may have heard about [the second ROOT Hackathon](https://indico.cern.ch/event/1463778/), that took place at IdeaSquare, CERN, last week! This in-person event was held on November 25-27. After a few days of recovering our energies, it's time to look back at this vibrant three days event, so let's go. 
+
+<center>
+    <img
+    src="{{'/assets/images/2nd-hackathon-intro.jpg' | relative_url}}"
+    alt="Hackathon opening."
+    style="width: 70%" width="1600" height="749" />
+</center>
+
+### The heart: participants
+
+This second hackathon vibrated with positive energy, fueled by a motivated group of 12 ROOT team members and 25 participants, many of which from CERN (thanks!). A standout achievement was our strides towards networking and team building, resulting in participation of people from across Europe (Italy, Czech republic, Spain...) and beyond (one participant from Qatar!), regardless of their gender, age, or background. The attendees could join three teams with different working topics, namely Python interfaces, Python Documentation and Tutorial modernisation, each guided by two coordinators from the ROOT team. This event indeed required a bit of organising and planning to put participants in condition to give their best. However, definitively worth the investment, we got a lot out of this event! 
```

Comment:
```suggestion
This second hackathon vibrated with positive energy, fuelled by a motivated group of 12 ROOT team members and 15 participants from outside the team. Many of the external participants work at CERN but a few participants came to CERN from across Europe (Italy, Czech Republic, Spain) and beyond (one participant from Qatar!) only for this event - some of them have used and contributed to ROOT since many years but for some this was the first time opening a PR - many thanks to all of you! 

The attendees could join three teams with different working topics, namely, Python Interfaces, Python Documentation and Tutorial Modernisation, each guided by two coordinators from the ROOT team.
```